includes/definitions/vertiv-pdu.yaml

```diff
@@ -5,6 +5,7 @@ icon: vertiv
 group: vertiv
 discovery:
     - sysObjectID:
+        - .1.3.6.1.4.1.21239.42.1.52
```

Comment:
Just these two Vertiv PDU devices?  or should the sysObjectID be less specific?

Reply:
We have only these two types of Vertiv Geist PDUs. With the sysObjectID - .1.3.6.1.4.1.21239.42.1.53 LibreNMS recognizes only one type. The second type needs the sysObjectID -.1.3.6.1.4.1.21239.42.1.52

Reply:
Yes, but `.1.3.6.1.4.1.21239.42.1.` would detect both of those.  Adding devices individually is a losing game.

Reply:
Ok i will try it on our test system this night.

Reply:
Both devices are recognized, no problems so far.